### ISSUE TITLE
Could not create task ':react-native-gesture-handler:compileDebugKotlin'

### ISSUE THREAD

**Issue by redundancer** [react-native-community/upgrade-support#251]
## Environment
```
System:
  OS: Linux 6.2 Ubuntu 22.04.3 LTS 22.04.3 LTS (Jammy Jellyfish)
  CPU: (12) x64 AMD Ryzen 5 1600 Six-Core Processor
  Memory: 20.61 GB / 31.29 GB
  Shell:
    version: 5.1.16
    path: /bin/bash
Binaries:
  Node: Not Found
  Yarn: Not Found
  npm: Not Found
  Watchman: Not Found
SDKs:
  Android SDK: Not Found
IDEs:
  Android Studio: Not Found
Languages:
  Java:
    version: 11.0.20.1
    path: /usr/bin/javac
  Ruby: Not Found
npmPackages:
  "@react-native-community/cli": Not Found
  react:
    installed: 18.2.0
    wanted: 18.2.0
  react-native:
    installed: 0.72.7
    wanted: ^0.72.0
npmGlobalPackages:
  "*react-native*": Not Found
Android:
  hermesEnabled: true
  newArchEnabled: false
iOS:
  hermesEnabled: Not found
  newArchEnabled: Not found

```

## Things I’ve done to figure out my issue

- [ ] I used [upgrade-helper](https://react-native-community.github.io/upgrade-helper/) to do my upgrade.

## Upgrading version

0.72

## Description

```
* What went wrong:
Could not determine the dependencies of task ':react-native-gesture-handler:bundleLibCompileToJarDebug'.
> Could not determine the dependencies of null.
   > Could not create task ':react-native-gesture-handler:compileDebugKotlin'.
      > Cannot use @TaskAction annotation on method AbstractKotlinCompile.execute() because interface org.gradle.api.tasks.incremental.IncrementalTaskInputs is not a valid parameter to an action method.

```

## Reproducible demo
Sorry I don't know how to reproduce this. Probably I missed something during the upgrade or so...

**Comment by lailton-b** [comment]
I'm facing the same problem, were you able to solve it?

**Comment by DejanKostyszyn** [comment]
Probably not. We stopped using react native at some point. It was just too annoying, even with basic tasks like upgrading...

**Comment by redundancer** [comment]
Probably not. We stopped using react native at some point. It was just too annoying, even with basic tasks like upgrading...

**Comment by redundancer** [comment]
If I remember correctly we had to set up the whole project again with the new version and then copy all our components from the old project. Sorry that I cannot help. I suggest you move to Flutter.

**Comment by lailton-b** [comment]
> If I remember correctly we had to set up the whole project again with the new version and then copy all our components from the old project. Sorry that I cannot help. I suggest you move to Flutter.

I have been thinking about that approach quite a bit... I really appreciate your replies, thanks a lot!

**Comment by redundancer** [comment]
Of course! React Native is not stable at all. We ran into so many problems that creating the apps with Java and Swift would have probably taken less time in the end. Not even very basic functionality like camera access is supported by the react native developers, yet. Hence, they were taken over by community volunteers who implement many bugs. Installing components crashed everything soo many times and no matter what we developed it almost always just worked on a subset of our mobile test devices. Really often there were crashes which we could not avoid, because they came from inside of fundamental components. Everything looks nice in emulators and simulators, but simply does not work on many real devices... Looks like my anger about react native introduces some bias in my responses haha
